enterprise/server/remote_execution/containers/ociruntime/ociruntime_test.go

```diff
@@ -17,7 +23,31 @@ import (
 	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
 )
 
+// Returns a special image ref indicating that a busybox-based rootfs should
+// be provisioned using the 'busybox' binary on the host machine.
+// Skips the test if busybox is not available.
+// TODO: use a static test binary + empty rootfs, instead of relying on busybox
+// for testing
+func manuallyProvisionedBusyboxImage(t *testing.T) string {
+	if _, err := exec.LookPath("busybox"); err != nil {
+		t.Skipf("skipping test due to missing busybox: %s", err)
+	}
+	return ociruntime.TestBusyboxImageRef
+}
+
+// Returns an image ref pointing to a remotely hosted busybox image.
+// Skips the test if we don't have mount permissions.
+// TODO: support rootless overlayfs mounts and get rid of this
+func realBusyboxImage(t *testing.T) string {
+	if !hasMountPermissions(t) {
+		t.Skipf("using a real container image with overlayfs requires mount permissions")
+	}
+	return "mirror.gcr.io/library/busybox"
+}
```

Comment:
This setup is pretty clunky but I wanted to make sure we could at least run some of the tests locally to make iteration a little faster until we figure out full rootless support. I don't think rootless support needs to be included with the initial release of this isolation type, though.